File: csharp/byps/BSkeleton.cs
/* USE THIS FILE ACCORDING TO THE COPYRIGHT RULES IN LICENSE.TXT WHICH IS PART OF THE SOURCE CODE PACKAGE */
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace byps
{
    public class BSkeleton : BRemote
    {
	    private BTargetId targetId;
	
	    public BTargetId BRemote_getTargetId() {
            return targetId;
	    }

	    public void BSkeleton_setTargetId(BTargetId targetId) {
            this.targetId = targetId;
	    }


    }
}

File: java/byps/src/byps/BSkeleton.java
package byps;

/* USE THIS FILE ACCORDING TO THE COPYRIGHT RULES IN LICENSE.TXT WHICH IS PART OF THE SOURCE CODE PACKAGE */

import byps.BRemote;
import byps.BTargetId;

public class BSkeleton implements BRemote {
  
  private BTargetId targetId;

	@Override
	public BTargetId BRemote_getTargetId() {
		return targetId;
	}

	void BSkeleton_setTargetId(BTargetId targetId) {
	  this.targetId = targetId;
	}
	
}
